app/scripts/lib/fxa-client.js

```diff
@@ -571,7 +571,8 @@ define(function (require, exports, module) {
                 return client.certificateSign(
                   sessionToken,
                   pubkey,
-                  duration);
+                  duration,
+                  { service: 'content-server' });
```

Comment:
I'm happy to. There's also a constant defined at the top of this module, which point would you rather define it at?